src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const img = document.querySelector('img.spider');
+const div = document.querySelector('div, wall');
```

Comment:
The selector 'div, wall' seems incorrect. If you intend to select a div with a class or ID of 'wall', you should use 'div.wall' or '#wall'.